packages/devextreme-angular/src/common/index.ts

```diff
@@ -74,6 +74,7 @@ export {
     VerticalEdge,
 } from 'devextreme/common';
 
+export * as AiIntegration from './ai-integration';
```

Comment:
No need to pay attention to this, it's internal auto-generated code. This type is eventually named AIIntegration.